info/guide/guide-chapter-3.md

```diff
@@ -379,11 +451,14 @@ fun foo() = goo().write("TooLong")
 ```
 **Valid example:**
 ```kotlin
-fun foo() = 
+fun foo() =
     goo().write("TooLong")
 ```
 
-Long `binary expression` should be split:
+`Long binary expression` should be split into priorities:
+1. Logic Binary Expression (`&&` , `||`)
+2. Comparison Binary Expression (`>` , `<` , `==` , `>` , `=` , `<=` , `!=`)
+3. Other types (Arithmetical and Bit operation) (`+` , `-` , `*` , `/` , `%` , `>>` , `<<` , `*=` , `+=` , `-=` , `/=` , `%=` , `++` , `--` , `in` , `!in` etc)
```

Comment:
bitwise/bitshift

Reply:
Done [78342c5](https://github.com/saveourtool/diktat/pull/1384/commits/78342c53dd92a42802f781323a6cc10b03605118)

Reply:
Add in Comparison Binary Expression `===` and `!==`